yantra/manager.py

```diff
@@ -74,7 +74,7 @@ def get_plugins(self):
                 # load the module and look for classes
                 module = imp.load_module(modname, fp, path, desc)
             except Exception as e:
-                error_msg = e.__class__.__name__ + ': ' + e.message
+                error_msg = e.__class__.__name__ + ': ' + str(e)
```

Comment:
If we add six as a dependency we can do:

```
error_msg = "{exception}: {message}".format(
                       exception=e.__class__.__name__,
                       message=six.text_type(e))
```

Reply:
I've implemented the first suggestion on the basis that we're expecting the yantra-dependent codebases to be upgraded to Python 3 in the near future. 

The second option requires slightly more work to implement and tear down once not required anymore. This is mainly because `six` was not available as a pip package and so we'd have to keep manual bookkeeping of the package

Reply:
Cool. We should swap our the str() call then with the unicode call.

I'd also recommend still swapping out the format string vs the '+' concatenation as it's a bit nicer, even just as a:
```
"{}: {}".format(e.__class__.__name__, unicode(e))
```

Reply:
That's a good point - I've made the changes accordingly